site2/docs/functions-package.md

```diff
@@ -449,34 +360,30 @@ This example demonstrates how to package a function in Go.
     }
     ```
 
-    > **Tip**
-    > 
-    > You can use context to connect with the Go function.
-    >
-    > ```
-    > if fc, ok := pf.FromContext(ctx); ok {
-    >    fmt.Printf("function ID is:%s, ", fc.GetFuncID())
-    >    fmt.Printf("function version is:%s\n", fc.GetFuncVersion())
-    > }
-    > ```
-
-    > **Note**
-    >
-    > - In `main()`, you **only** need to register the function name to `Start()`. **Only** one function name can be received in `Start()`. 
-    >
-    > - Go function uses Go reflection based on the received function name to verify whether the parameter list and returned value list implemented are correct. The parameter list and returned value list specified **must be** one of the following sample functions:
-    >
-    >   ```
-    >   func ()
-    >   func () error
-    >   func (input) error
-    >   func () (output, error)
-    >   func (input) (output, error)
-    >   func (context.Context) error
-    >   func (context.Context, input) error
-    >   func (context.Context) (output, error)
-    >   func (context.Context, input) (output, error)
-    >   ```
+    You can use context to connect to the Go function.
+    
+    ```
+    if fc, ok := pf.FromContext(ctx); ok {
+        fmt.Printf("function ID is:%s, ", fc.GetFuncID())
+        fmt.Printf("function version is:%s\n", fc.GetFuncVersion())
+     }
+     ```
+
+    When writing a Go function, remember that
+    - In `main()`, you **only** need to register the function name to `Start()`. **Only** one function name is received in `Start()`. 
+    - Go function uses Go reflection based on the received function name to verify whether the parameter list and returned value list implemented are correct. The parameter list and returned value list specified **must be** one of the following sample functions:
```

Comment:
I think this sentence needs to be refined:
whether the parameter list and returned value list implemented are correct.
The parameter list and returned value list specified **must be** one of the following sample functions:

Reply:
nice catch, I can simplify it, thanks for your reminder.